avaframe/out3Plot/outAIMEC.py

```diff
@@ -158,7 +158,7 @@ def visuRunoutComp(rasterTransfo, resAnalysis, newRasters, cfgSetup, cfgPath, cf
     pU.putAvaNameOnPlot(ax, projectName)
 
     outFileName = '_'.join([projectName, resType,
-                            'thresholdValue', str(int(thresholdValue)), 'slComparison'])
+                            'thresholdValue', str(thresholdValue).replace('.', ''), 'slComparison'])
```

Comment:
this might be then hard to read the number.. maybe dont use the decimals or separate by underline?